articles/other/other-how-use-pricing-tool.md

```diff
@@ -0,0 +1,118 @@
+---
+title: How to use the UKCloud Pricing Estimator Tool | UKCloud Ltd
+description: Describes how to use the UKCloud Pricing Estimator Tool
+services: other
+author: Dan Baker
+
+toc_rootlink: How To
+toc_sub1: 
+toc_sub2:
+toc_sub3:
+toc_sub4:
+toc_title: Use the UKCloud Pricing Estimator Tool
+toc_fullpath: How To/other-how-use-pricing-tool.md
+toc_mdlink: other-how-use-pricing-tool.md
+---
+
+# How to use the UKCloud Pricing Estimator Tool
+
+## Overview
+
+The UKCloud Pricing Estimation Tool has been created to assist in the cost estimation of our core G-Cloud 10 services. Refer to the G-Cloud 10 Pricing Guide for information regarding service options, pricing options, any required set up fees and product rules. To achieve the most accurate estimate please consult with one of our Cloud Architects.
+
+> [!IMPORTANT]
+> You should always ensure that you're using the latest version of the pricing estimation tool by checking on the [UKCloud web site](https://ukcloud.com/price-estimator-tool/). Also note that this tool provides an estimate based on the selections you make. Your actual consumption of the service may be different, resulting in higher or lower actual costs.
+
+> [!NOTE]
+> There are some known bugs in the latest versions of the tool. For more information see [Known bugs with the UKCloud Pricing Estimator Tool](pricing-estimator-bugs.md)
+
+## Installing the tool
+
+The Windows version of the tool is provided as a  `.exe` file and the MacOS version is packaged as a `.dmg` file.
+
+Once you've downloaded the install file from the UKCloud web site, depending on the version of the tool you're using, you may to install it first.
```

Comment:
you may need to install it first

Reply:
The windows version downloads as a straight .exe file. Requires no install. The Mac is a .dmg file and you open it and copy it to the 'Applications' folder. So I think this is fine.

Reply:
@SteveHallUKCLoud I've made some changes based on your comments